stdlib/asm/crypto/fri/frie2f4.masm

```diff
@@ -166,6 +178,9 @@ end
 
 #! Verifies one FRI query.
 #!
+#! This procedure is specialized to the case when the remainder polynomial, used in the final check,
+#! is expected to have degree at most 64.
+#!
 #! Input:  [poe, p, e1, e0, layer_ptr, rem_ptr, ...]
 #! Output: [x, x, x, x, x, x, x, x, x, x, ...]
```

Comment:
```suggestion
#! Output: [X, x, x, x, x, x, x, x, x, ...] (12 "garbage" elements)
```

Reply:
Good point, fixed it but used small `x` to be consistent